vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
 library(malariasimulation)
 library(malariaEquilibrium)
-library(reshape2)
 ```
 
-# Parameterisation
+Long-lasting insecticidal bednets are highly effective and relatively cheap interventions to prevent malaria. The effects of insecticidal bed net distribution campsigns can be simulated using malariasimulation. The model provides the user with the flexibility to specify parameters that describe the net distribution campaign (e.g. timing, coverage, and target population), the bed nets distributed (e.g. efficacy, longevity), and the relative densities of the mosquito species present in the target population. We will illustrate this through an example with two bednet distributions, once per year.
 
-We are going to set the default parameters to run the simulation from an equilibrium.
+### Plotting functions
+We can create a few plotting functions to visualise the output.
+```{r}
+cols <- c("#E69F00", "#56B4E9", "#009E73", 
+          "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
+
+# Plotting functions
+plot_prev <- function() {
+  plot(x = output$timestep, y = output$n_detect_730_3650/output$n_730_3650, 
+       type = 'l', col = cols[3], lwd = 2.5,
+       xlab = 'Time (days)', ylab = expression(paste(italic(Pf),"PR"[2-10])),
+       xaxs = "i", yaxs = "i", bty = "l", ylim = c(0.15, 0.93))
+  lines(x = output_control$timestep, y = output_control$n_detect_730_3650/output_control$n_730_3650,
+        col = cols[5], lwd = 2.5)
+  abline(v = bednetstimesteps, col = 'black', lty = 2, lwd = 2.5)
+  grid(lty = 2, col = 'grey80', lwd = 1)
+  legend('bottomleft', box.lty = 0, legend = c('Bednets','Prevalence for bednet scenario','Prevalence for control scenario'),
+         col = c('black', cols[3], cols[5]), lty = c(2,1,1), lwd = 2.5)
+}
+```
+  
+### Parameterisation
+
+Use the `get_parameters()` function to generate the list of parameters for a seasonal profile, accepting the default values to run the simulation from an equilibrium starting point. The seasonality will drive mosquito dynamics in the absence of interventions. 
```

Comment:
Seasonality isn't the focus of this vignette so I think should be turned off by default